### ISSUE TITLE
Display portals at top of help Library

### ISSUE THREAD

**Issue by russ-manta** [MantaRayMedia/alnap#275]
### Done criteria
- [x] Modify portal content type to include link and image fields, and maybe card text
- [x] Modify Help library attachment view to display portals added, perhaps also add nodequeue to view

---
### Designs
- [ ] See help library designs

---
### ALNAP to check
- [ ] Check can add new portals to the top of the help library (an order using number field)
- [ ] Check portals display ok at top of help library

**Comment by russ-manta** [comment]
@cutlern please add nodequeue in the morning

**Comment by cutlern** [comment]
Using a select list field instead of nodequeue/entityqueue because they're not supported by search api

**Comment by Maria-alnap** [comment]
@russ-manta @matty-manta can you please explain to us how to add a portal to the top of the HELP search as mentioned above? No idea how to do it. Thanks!

**Comment by matty-manta** [comment]
@Maria-alnap Sure - we've added a new Portal content type to manage this.

If you head to the content table:

http://alnap-dev.mrmdev.co.uk/admin/content

And filter for Portal - you'll see that we've created two items.

If you edit the Urban humanitarian response item, you'll see how it works. 

There are fields to provide a link to the relevant portal, and then a field called "Order", which determines where it appears at the top of the Help library. The Urban humanitarian response appears first on the Help library, as the Order number is 01. 

Then just fill out the card content fields to control how it appears (image, summary) on the Help Library page.

Any qs just let me know :)

Matty
